linux_os/guide/system/software/integrity/fips/is_fips_mode_enabled/oval/shared.xml

```diff
@@ -0,0 +1 @@
+{{{ oval_file_contents(filepath='/proc/sys/crypto/fips_enabled',filepath_id='',contents='1') }}}
```

Comment:
Delete this file and use the `lineinfile` template as suggested in the rule.yml file

```suggestion
```

Reply:
This macro (`oval_file_contents`) is not really adapted to be used for general purposes, without `filepath_id` set, it would create some weird OVAL definition names.